test/1.4/errors/T_ESWITCH.dml

```diff
@@ -11,12 +11,6 @@ method init() {
         /// ERROR ESWITCH
     {
     }
-    switch (1)
-        // must start with a case
-        /// ERROR ESWITCH
-    {
-        ;
-    }
```

Comment:
Artifact from when the RAII parser changes were present. I thought I had unstaged this.